geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/AttributeIndexStrategyTest.scala

```diff
@@ -898,12 +898,18 @@ class AttributeIndexStrategyTest extends Specification with TestWithDataStore {
 
   "AttributeIndexLikeStrategy" should {
 
-    "correctly query on strings" in {
+    "correctly query on strings with multi-char match" in {
       val features = execute("name LIKE 'b%'")
       features must haveLength(2)
       features must contain("bill", "bob")
     }
 
+    "correctly query on strings with single-char match" in {
+      val features = execute("name LIKE 'b_b'")
+      features must haveLength(1)
+      features must contain("bob")
+    }
```

Comment:
Should we add a word like 'bobbed' to the test to show that an underscore doesn't match multi-chars?

Reply:
sure, I'll add it to the one in index-api, so I don't have to update all the other tests in this class.